README.md

```diff
@@ -57,6 +57,36 @@ const MyComponent = props => {
 export default MyComponent;
 ```
 
+### Next.js 13
+
+In Next.js 13, you need to use the new `use client` declarative.
+
+```jsx
+// components/Editor.js
+'use client'
+import SunEditor from 'suneditor-react'
+import 'suneditor/dist/css/suneditor.min.css'
+                                           
+export default function Editor() {
```

Comment:
Shouldn't we accept the SunEditor props and spread it in `<SunEditor />`?

Reply:
Fixed that.